cmd/publishing-bot/main.go

```diff
@@ -92,6 +96,10 @@ func main() {
 		glog.Fatalf("Target organization cannot be empty")
 	}
 
+	if len(cfg.RulesFile) == 0 && len(os.Getenv("RULE_FILE_PATH")) == 0 {
```

Comment:
the env var reading should also be here. Less magic is better.

Reply:
Do we actually need this env var? Can we drop it? We don't have env vars for the other options either.

Reply:
@sttts we need the baseRepoPath, but I think we can construct it here